src/main/resources/application.yml

```
@@ -15,4 +15,8 @@ management:
 okta:
   oauth2:
     issuer: ${OKTA_ISSUER:https://dev-18092578.okta.com/oauth2/default}
-    audience: ${OKTA_AUDIENCE:api://default}
\ No newline at end of file
+    audience: ${OKTA_AUDIENCE:api://default}
+    
+client.vsac_base_url: https://vsac.nlm.nih.gov/vsac
+client.service_ticket_endpoint: /ws/Ticket/%s?service=http://umlsks.nlm.nih.gov
+client.valueset_endpoint: /svs/RetrieveMultipleValueSets?id={oid}&ticket={st}&profile={profile}&includeDraft={includeDraft}
```

```suggestion
client:
    vsac_base_url: https://vsac.nlm.nih.gov/vsac
    service_ticket_endpoint: /ws/Ticket/%s?service=http://umlsks.nlm.nih.gov
    valueset_endpoint: /svs/RetrieveMultipleValueSets?id={oid}&ticket={st}&profile={profile}&includeDraft={includeDraft}
```
See if these still work with the YAML organization style.

---

@jkotanchik-SB You are right, it's better styling and works. I just pushed in the change. Thanks Joe!